1.4/Patches/Mods/Viles Materials.xml

```diff
@@ -15,8 +15,8 @@
 				<xpath>Defs/ThingDef[defName="Yautjavium"]</xpath>
 				<value>
 					<li Class="Celsius.StuffThermalProperties">
-						<volumetricHeatCapacity>800</volumetricHeatCapacity>
-						<insulationFactor>0.17</insulationFactor>
+						<volumetricHeatCapacity>1000</volumetricHeatCapacity>
```

Comment:
I will leave all these to your judgement, as im not completely sure the amterials youve based them off, i mainly did it based on the temperature and hit points stats, as well as averaged them out between the amteraisl they are made from.